src/NosCore.Controllers/Miniland/MlobjPacketHandler.cs

```diff
@@ -32,6 +33,7 @@ namespace NosCore.PacketHandlers.Miniland
     public class MlEditPacketHandler : PacketHandler<MLEditPacket>, IWorldPacketHandler
     {
         private readonly IMinilandProvider _minilandProvider;
+        private readonly IFriendHttpClient _friendHttpClient;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private field '_friendHttpClient'.](https://app.codacy.com/app/NosCoreIO/NosCore/pullRequest?prid=4513002)